web_development_101/installations/installation_overview.md

```diff
@@ -36,7 +36,7 @@ If you're using a Mac, you're in great shape. Most professional developers rely
 
 #### Linux
 
-[Linux](https://en.wikipedia.org/wiki/Linux) is a free and open-source operating system that works well with all programming languages. Most development tools are written to work natively with Linux. Your tools will likely be updated more often, have more information available for troubleshooting, and just plain run better on Linux. We'll be using Ubuntu, one of the most popular and user-friendly versions available. **If you don't use a Mac, we recommend that you use Linux.** It's that simple.
+[Linux](https://en.wikipedia.org/wiki/Linux) is a free and open-source operating system that works well with all programming languages. Most development tools are written to work natively with Linux. Your tools will likely be updated more often, have more information available for troubleshooting, and just plain run better on Linux. We'll be using Ubuntu, one of the most popular and user-friendly versions available. **If you don't use a Mac, we recommend that you use Linux.** It's that simple. If you opt to use Linux on a computer with 4gb RAM or less, it's best to use a light-weight version of Ubuntu to avoid lag or even freezing of the system. For more information on light-weight options, refer to this [table](https://www.reddit.com/r/Ubuntu/comments/8fz0dv/disk_and_ram_usage_of_ubuntu_kubuntu_lubuntu/dy7iye5/).
```

Comment:
We already recommend Xubuntu both for VMs (https://github.com/TheOdinProject/curriculum/blob/3a51b15fe0e15fbad7524b96536307117c4484a2/web_development_101/installations/prerequisites.md#step-12-download-linux) and for dual boots on older computers (https://github.com/TheOdinProject/curriculum/blob/3a51b15fe0e15fbad7524b96536307117c4484a2/web_development_101/installations/prerequisites.md#step-1-download-linux). 

Perhaps that would be a better location to add the specific recommendation for computers with 4 GB of RAM of less?
If you'd like to also add something here, then I'd suggest keeping it simple, e.g., "We'll be using Ubuntu, one of the most popular and user-friendly versions available, or the lighter weight alternative Xubuntu."

Reply:
Back when I did the set up process, I completely skipped the VM and dual boot descriptions, because I decided to format my hard drive and do a fresh linux install. That's why I'd like to add something there.

Would this be okay? "We'll be using Ubuntu, one of the most popular and user-friendly versions available, or a lighter weight alternative based on Ubuntu, such as Xubuntu." 

I basically would like the user to know that he can feel save to use a distribution based on Ubuntu.